tests/test_encode_decode.py

```diff
@@ -5,15 +5,17 @@
 import io
 import os
 import platform
+import plistlib
 import quopri
 import string
 import sys
 import unittest
 import uu
 from tempfile import TemporaryDirectory
 
-from hypothesis import example, given, strategies as st, target
+from hypothesis import HealthCheck, example, given, settings, strategies as st, target
 
+no_health_checks = settings(suppress_health_check=HealthCheck.all())
```

Comment:
Let's apply `@settings(suppress_health_check=HealthCheck.all())` directly to our test method, instead of having a module constant like this.